weaviate/data/crud_data.py

```diff
@@ -646,6 +672,8 @@ def delete(
             by default None
         consistency_level : Optional[ConsistencyLevel], optional
             Can be one of 'ALL', 'ONE', or 'QUORUM'. Determines how many replicas must acknowledge
+        tenant_key: str on None, optional
+            Defined in multitenancy config tenantKey setting.
```

Comment:
it needs a new line after this one:
```
        tenant_key: str on None, optional
            Defined in multitenancy config tenantKey setting.

        Examples
        --------
```

otherwise the docsting on readthedocs will look off.